src/miner_app.erl

```diff
@@ -30,6 +30,20 @@ start(_StartType, _StartArgs) ->
                       BBOpts ++ [{block_cache, Cache}]}],
     application:set_env(rocksdb, global_opts, Opts),
 
+    HotfixDir = application:get_env(miner, hotfix_dir, "hotfix"),
```

Comment:
Should the default be `"/opt/miner/hotfix"` here?

Reply:
I'm not sure there needs to be a default.

Reply:
in any case i was trying to follow the pattern we have for `BaseDir` where we default the value to `"data"` and then in most of the configs and run scripts override it to `"/var/data"`

Reply:
> in any case i was trying to follow the pattern we have for `BaseDir` where we default the value to `"data"` and then in most of the configs and run scripts override it to `"/var/data"`

I think that makes sense so in a dev/local env the `hotfix` dir will be just next to data presumably. That's probably okay.